.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '1.14'
+    - uses: actions/checkout@v2
+    - uses: actions/cache@v2
+      with:
+        path: |
+          ~/go/pkg/mod
+          ~/go/pkg/sumdb
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+    - name: Install tools
+      env:
+        GO111MODULE: 'on'
+      run: |
+        cd
+        go get \
+          gotest.tools/gotestsum \
```

Comment:
we should consider adding these to `tools.go` in the root so they are tracked with `go.mod`.

Reply:
Ah yeah, I forgot about that pattern!

Reply:
:^)